public/Update-DbaBuildReference.ps1

```diff
@@ -53,16 +53,28 @@ function Update-DbaBuildReference {
                 $EnableException
             )
             $url = Get-DbatoolsConfigValue -Name 'assets.sqlbuildreference'
+            $proxyUrl = Get-DbatoolsConfigValue -FullName network.proxy.url
+            $proxyUsername = Get-DbatoolsConfigValue -FullName network.proxy.username
+            $proxySecurePassword = Get-DbatoolsConfigValue -FullName network.proxy.securepassword
+            if ($proxyUsername) {
+                $proxyCredential = New-Object System.Management.Automation.PSCredential ($proxyUsername, $proxySecurePassword)
+            }
+
```

Comment:
Yeah I'm more for this being embedded into `Invoke-TlsWebRequest` instead of having to copy this everywhere